antidox/setup.py

```diff
@@ -0,0 +1,80 @@
+#
```

Comment:
Can you explain why this file is needed or how it's used?  It doesn't seem like it's imported by perspective.py

Reply:
it used used has all the requirements for the distributed systems.

--setup_file ./setup.py